### PR TITLE
fixed logger usage (no concat, nor object alloc) + code formatting

### PR DESCRIPTION
**Pull request by bourgesl**

### CODE REVIEW

**Review by jfree — APPROVED** [review]

File: src/main/java/org/jfree/skija/SkijaGraphics2D.java [hunk 1]
```diff
@@ -313,22 +385,26 @@ private Path path(Shape shape) {
...
+            LOGGER.debug("draw(Shape) : {}", s);
+        }
         this.skijaPaint.setMode(PaintMode.STROKE);
         if (s instanceof Line2D) {
             Line2D l = (Line2D) s;
             this.canvas.drawLine((float) l.getX1(), (float) l.getY1(), (float) l.getX2(), (float) l.getY2(), this.skijaPaint);
         } else if (s instanceof Rectangle2D) {
             Rectangle2D r = (Rectangle2D) s;
-            if (r.getWidth() < 0.0 || r.getHeight() < 0.0) {
+            if (r.getWidth() <= 0.0 || r.getHeight() <= 0.0) {
```
Comment (lukeu) [inline src/main/java/org/jfree/skija/SkijaGraphics2D.java:397]: Might have been an improvement to split behavioural changes into a separate commit so the history is clearer to follow(?) The commit one-line talks only about non-functional (performance) changes and formatting. (Tip: the one thing I find `git gui` super useful for is splitting a prepared commit up into 2 or more parts.)

Comment (bourgesl) [reply src/main/java/org/jfree/skija/SkijaGraphics2D.java:397]: Other PRs in prep...
